demisto_sdk/commands/lint/resources/pylint_plugins/base_checker.py

```diff
@@ -44,12 +43,13 @@ def _print_checker(self, node):
         except Exception:
             pass
 
-    def _sys_exit_checker(self, node):
-        try:
-            if node.func.attrname == 'exit' and node.func.expr.name == 'sys' and node.args and node.args[0].value != 0:
-                self.add_message("sys-exit-exists", node=node)
-        except Exception:
-            pass
+    def _sleep_checker(self, node):
```

Comment:
if someone `from time import sleep` he can only have `sleep(x)` in the code, let's make sure we cover this as well.

Reply:
you are right , added a check for that and a test.